.azure/modules/functionApp/slackNotifier.bicep

```
@@ -158,7 +158,7 @@ resource notifyDevTeam 'Microsoft.Insights/actionGroups@2023-01-01' = {
 var query = '''
             union
                  (exceptions
-                 | where problemId != "ClientConnectionFailure at forward-request" and problemId != "ClientConnectionFailure at transfer-response"),
+                 | where not(type == "ClientConnectionFailure" and customDimensions["Service ID"] == "altinn-dev-api")),
```

Are we getting any noise from tt02 as well? Maybe add something like this if so? 
I'm guessing we don't get anything from prod yet 🙈 

```suggestion
                 | where not(type == "ClientConnectionFailure" and (customDimensions["Service ID"] == "altinn-dev-api" or customDimensions["Service ID"] == "altinn-tt02-api")),
```

---

That is a good point. On the other hand, we haven't seen these errors from tt02 yet, and I'd be very interested to know if/when they start popping up. I'm hoping that some of the errors we are seeing are due to the cheaper SKU on the dev apim and maybe on our components as well. I suggest we let errors from tt02 appear in the alerts channel for now, and filter them out if they start to become a problem.

---

🚀